spec/rental_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+
+describe Rental do
+  before(:each) do
+    @book = Book.new('Tell Me Your Dreams', 'Sidney Sheldon')
+    @person = Person.new(7, 'Syrs', parent_permission: true)
+    @rental = Rental.new('20/02/2022', @person, @book)
+  end
+
+  it 'Create a new rental with a date, book and person' do
+    expect(@rental).to have_attributes(date: '20/02/2022')
+  end
+  it 'Object instance of rental' do
+    expect(@rental).to be_an_instance_of Rental
+  end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_person and add_book` class rental method has (at least) an 'it' block 👇🏻 check your book.rb file

```ruby
   def add_person(person)
    @person = person
    person.rentals_list.push(self) unless person.rentals_list.include?(self)
  end

  def add_book(book)
    @book = book
    book.rentals_list.push(self) unless book.rentals_list.include?(self)
  end
  end
```

Reply:
Already implemented it! Thanks for the feedback! :smile: